### ISSUE TITLE
Load testing - Send k6 results to prometheus

### ISSUE THREAD

**Issue by lmoesle** [it-at-m/digiwf-project#81]

**Comment by lmoesle** [comment]
K6 supports pushing its metrics to prometheus -> https://k6.io/docs/results-visualization/prometheus/

Currently, we use `ServiceMonitor`s for promethteus to poll metrics from applications with http endpoints. We have to check if pushing metrics is possible in the cap. We also have to check if the currently running prometheus is able to deal with the load produced by the k6 loadtests pushing metrics every 1s.

**Comment by lmoesle** [comment]
Discussion with @dominikhorn93 :

It's not possible to push metrics to prometheus in the cap. We need to find away for prometheus to poll k6s loadtesting metrics. 

We want to trigger loadtests manually. Our approach with a `Deployment` object is suitable for now. If we want to start loadtests we scale up the pods of the deployment and if we are done we scale it back down to 0.
Another option for the future is to execute the pod only once by using e.g. a `Job` object (https://docs.openshift.com/container-platform/4.7/nodes/jobs/nodes-nodes-jobs.html).

**Comment by lmoesle** [comment]
Won't fix
